src/components/header/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { ReactComponent as ProfileIcon } from '../../assets/profile.svg';
+import { ReactComponent as RingIcon } from '../../assets/ring.svg';
+import { ReactComponent as SearchIcon } from '../../assets/search.svg';
+import { links } from '../navBar/helper';
+import { useLocation } from 'react-router-dom';
+import {
+  HeaderContainer,
+  IconsContainer,
+  PhotoContainer,
+  ProfileContainer,
+  StyledHeaderText,
+  StyledHr,
+  StyledTypography,
+} from './styled';
+
+const Header: React.FC = () => {
+  const location = useLocation();
+  const currentLink = links.find((link) => link.to === location.pathname);
+  const headerText = currentLink ? currentLink.name : links[0].name;
```

Comment:
What is a case when currentLink is undefined?

Reply:
I am not sure about this one. In the previous review i had 'Overview' as a default in place of 'links[0].name', but you said to change it. Maybe we should have some general default value here, like "Heading" ? or 'Unknown" or it should show up at all. 
fix: changed to empty place with space ' '

Reply:
![image](https://github.com/VladyslavGilyuk/chi-pet-project/assets/68470050/f9cffc90-b2f1-4b31-a64a-66255a6c6d5c)

Reply:
fix: Coreccted